pkg/logging/sentry_hook.go

```diff
@@ -35,15 +26,27 @@ func (hook *SentryHook) Fire(entry *logrus.Entry) error {
 	if hub == nil {
 		hub = sentry.CurrentHub()
 	}
+	client, scope := hub.Client(), hub.Scope()
+	if client == nil || scope == nil {
+		return ErrorHubInvalid
+	}
```

Comment:
Is it still possible to run Poseidon without Sentry (if no DSN is set)? Sentry should always remain an optional component.

Reply:
True! Yes, Poseidon still runs without error of no DSN set.
This works because Sentry handles the empty DSN transparently (using its `noopTransport`). The `ErrorHubInvalid` should never be thrown, as the scope is initialized by Sentry and we initialize the client on Poseidons startup.